sass/components/_popover.scss

```diff
@@ -67,7 +67,12 @@
     }
 
     &.bottom,
-    &.right,
+
+    &.right {
```

Comment:
I think this might result in similar behaviour to the things I've commented on in your other two PRs. Unless we can just set a `border-color` for all of these rules, we'll have to add separate rules for `.bottom`, `.right`, and so on.

Reply:
Yup totally, I've updated all 4 PRs to place the CSS styles in the correct place. Thank you for the sharp spotting!